omnibus/package-scripts/agent-dmg/postinst

```diff
@@ -107,18 +107,23 @@ if [ ! -e "$CONF_DIR/datadog.yaml" ]; then
     exit 1
 fi
 
-# Start the app only if it's not a script install
-if [ -z "$SCRIPT_INSTALL" ]; then
-    echo "# Starting the app"
-    # -a for application, -F for fresh, do not restore old app
-    TMPDIR=$(sudo -u "$INSTALL_USER" getconf DARWIN_USER_TEMP_DIR)
-    export TMPDIR
-    sudo -u "$INSTALL_USER" open -Fa 'Datadog Agent'
+# Start the tray icon app only when not in system-wide installation
+if [ ! -f "/tmp/install-ddagent/system-wide" ]; then
+  # Start the app only if it's not a script install
+  if [ -z "$SCRIPT_INSTALL" ]; then
+      echo "# Starting the app"
+      # -a for application, -F for fresh, do not restore old app
+      TMPDIR=$(sudo -u "$INSTALL_USER" getconf DARWIN_USER_TEMP_DIR)
+      export TMPDIR
+      sudo -u "$INSTALL_USER" open -Fa 'Datadog Agent'
+  fi
```

Comment:
Should we explicitly remove the `system-wide` file after a successful install? Even though I doubt this will change much in most cases, if the user was to reinstall the agent as system-wide a few minutes later, the file would still be present and would cause the postinst to behave as a local install

Reply:
Good point 👍 It will be safer overall to remove these files